static/bidder-info/aidem.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "https://zero.aidemsrv.com/ortb/v2.5/bid/request"
+maintainer:
+  email: prebid@aidem.com
+modifyingVastXmlAllowed: true
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+      - native
+  site:
+    mediaTypes:
+      - banner
+      - video
+      - native
+userSync:
+  redirect:
```

Comment:
url is not redirecting 
https://gum.aidemsrv.com/prebid_sync?gdpr=&consent=&us_privacy=&redirect=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Daidem%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24UID

![image](https://github.com/prebid/prebid-server/assets/87759626/d91ffbc0-03b7-4827-ac20-21a229cf6806)

Reply:
In case no userID can be provided (e.g. in this case the GDPR consent was invalid) we return a 204. Is this the correct procedure? We didn't find any details on the docs.

Reply:
@Sonali-More-Xandr Hello Sonali, do you have any feedback on this? Thanks.

Reply:
@gsollazzo Thank you for your comment. In order for us to validate a valid user sync URL, we would need to check if the URL is being properly redirected. Could you please help in providing a URL that actually works so that we can test that `UID` macro is properly set in the response?

Reply:
@gargcreation1992 we can! Are you based in EU or US? We will add valid TCF/USP values to the URL.

Reply:
We can test it from US location.

Reply:
**NOTICE**: By adding `us_privacy=1YNN` you grant AIDEM permission to track your browser.
You can use this URL [https://gum.aidemsrv.com/prebid_sync?gdpr=&consent=&us_privacy=1YNN&redirect=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Daidem%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24UID](https://gum.aidemsrv.com/prebid_sync?gdpr=&consent=&us_privacy=1YNN&redirect=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Daidem%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24UID)

Reply:
thanks! We are now able to validate that the ^^ user sync URL works.